node/lib/TranslatorInfo.ts

```diff
@@ -0,0 +1,26 @@
+
+/**
+ * Information about a translator, as loaded from the translator's manifest.
+ */
+export class TranslatorInfo {
+    /**
+     * Fully-qualified name of the translator (required).
+     */
+    public readonly name: string;
+
+    /**
+     * List of interfaces implemented by the translator.
+     */
+    public readonly interfaceNames: string[];
```

Comment:
My naming convention has been that an interface "name" is the string identifier, while an "interface" or "deviceInterface" is an instance of the `DeviceInterface` class.

---
In reply to: [71904218](https://github.com/opent2t/opent2t/pull/1#discussion_r71904218) [](ancestors = 71904218)